components/selection/selection-info-controller.js

```diff
@@ -0,0 +1,51 @@
+export class SelectionInfoController {
+
+	static get properties() {
+		return {
+			keys: { type: Object },
+			state: { type: String }
+		};
+	}
```

Comment:
I'm not sure if these actually need to be defined as properties like this. All the Lit docs use TypeScript 😠